AppOne/AppOne/Model/SectionTitles.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  SectionTitles.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 21.11.22.
+//
+
+struct SectionTitles : Hashable {
```

Comment:
singular `SectionTitle`
```suggestion
struct SectionTitle : Hashable {
```